bi-connectors/TableauConnector/src/dialect.tdd

```diff
@@ -205,6 +246,22 @@
             <argument type='datetime' />
             <argument type='real' />
         </function>
+        <function group='operator' name='-' return-type='datetime'>
+            <formula>DATE_SUB(%1, INTERVAL %2 DAY)</formula>
+            <argument type='date' />
+            <argument type='int' />
+        </function>
+        <function group='operator' name='-' return-type='datetime'>
+            <!-- 86400 as it represents seconds in a day -->
+            <formula>DATE_SUB(%1, INTERVAL CAST((86400 * %2) AS INT) SECOND)</formula>
```

Comment:
What does the `%2` do?

Reply:
%1 is the first argument and %2 are the arguments in the following lines